src/ECharts/Options/Unsafe.purs

```diff
@@ -0,0 +1,129 @@
+module ECharts.Options.Unsafe(
```

Comment:
This whole module is extremely unsafe. Let's talk about ways to eliminate it or make it more type safe. I'm on IRC and Skype at jdegoes.